src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -44,10 +46,14 @@ def load_arguments(self, _):
         c.argument('admin_enabled', help='Indicates whether the admin user is enabled', choices=['true', 'false'])
 
     with self.argument_context('acr repository delete') as c:
-        c.argument('manifest', nargs='?', required=False, const='', default=None, help='The sha256 based digest of manifest to delete')
+        c.argument('manifest', nargs='?', required=False, const='', default=None, help=argparse.SUPPRESS)
```

Comment:
This will allow the user to actually supply `--manifest`.  Is that the intention?   If not, use `c.ignore('manifest')`.

Reply:
Yes we still want to support `--manifest` to avoid breaking users' current scripts. We will slowly remove the support in the future.